vm/systemSmartContracts/staking.go

```diff
@@ -513,6 +532,11 @@ func (r *stakingSC) stake(args *vmcommon.ContractCallInput, onlyRegister bool) v
 	}
 
 	if !onlyRegister {
+		if !r.canStake() {
+			r.eei.AddReturnMessage("staking is full")
```

Comment:
Test for this branch?

Reply:
a lot of tests has been changed to work because of this check.